resources/assets/ts/Utilities.ts

```diff
@@ -1,4 +1,6 @@
-enum Club {
+import * as $ from "jquery"
+
+export enum Club {
     None,
     BcClub = 2,
```

Comment:
Yeah good catch. This line just changed because of the new Tyepscript structure (imports), but I agree with you. Should probably not do this in this branch. I added a ticket (https://trello.com/c/IprS92RG/325-2-generalize-clientside-code-typescript),